### DISCUSSION TITLE
Human Tasks

### DISCUSSION THREAD

**Discussion by josesuero** [serverlessworkflow/specification#739] [Q&A]
We have a need to set a workflow that includes a combination of human and systems tasks, we're using Kogito already, so I thought it would make a great proof of concept to implement a serveless workflow.

The spec includes callbacks, which I guess can be used to implement human tasks, we could trigger an event or an rest endpoint to achieve this. 

But I haven't figure out a way to display this "pending" callbacks, other than keeping state myself. For reference I'd like the equivalent of a BPMN human task, where I can see what's holding the workflow and ask a user to fill it, ideally this would be stored somewhere to keep statistics, this works mostly "out-of-the-box" with bpmn and Kogito tasks api

How would I achieve this in serverless workflows?

Thanks in advance

**Comment by ricardozanini** [comment]
I believe this is more of an implementation detail than a spec concern. The "pending" callbacks should be handled by the implementation and offer a way for users to either query or view them somehow in the solution/UI. I believe tasks are yet another scope, and I'm unsure if we'd like to bring them here.

Although, I can see maybe an extension to the spec to classify a callback as a pending task.
